bot/cogs/snekbox.py

```diff
@@ -205,6 +205,12 @@ async def send_eval(self, ctx: Context, code: str) -> Message:
             if paste_link:
                 msg = f"{msg}\nFull output: {paste_link}"
 
+            # Collect stats of eval fails + successes
+            if icon == ":x:":
+                self.bot.stats.incr("snekbox.python.fail")
+            elif icon in (":warning:", ":white_check_mark:"):
```

Comment:
Safe to simplify this to an `else`.
```suggestion
            else:
```